addons/repair/functions/fnc_doRepair.sqf

```diff
@@ -31,9 +31,23 @@ _hitPointDamage = _hitPointDamage max ([_unit] call FUNC(getPostRepairDamage));
 // raise event to set the new hitpoint damage
 ["setVehicleHitPointDamage", _vehicle, [_vehicle, _hitPoint, _hitPointDamage]] call EFUNC(common,targetEvent);
 
+// Repair the rest in the group (no specific damage, main hitpoint is enough)
+_hitpointGroup = configFile >> "CfgVehicles" >> typeOf _vehicle >> QGVAR(hitpointGroup);
+_hitpointGroup = if (isArray _hitpointGroup) then {getArray _hitpointGroup} else {[]};
+if (count _hitpointGroup > 0) then {
+    ([_vehicle] call EFUNC(common,getHitPointsWithSelections)) params ["_hitpoints"];
+    _hitpointGroup deleteAt 0; // Remove main group hitpoint
+    {
+        if (_x in _hitpoints) then {
+            _vehicle setHitPointDamage [_x, 0];
```

Comment:
Besides the comment I made above; this will not work. Use `setVehicleHitPointDamage` targetEvent for this, becase the `setHitPointDamage` is best set locally.